src/python/pants/help/help_printer.py

```diff
@@ -293,8 +293,7 @@ def _print_target_help(self, target_alias: str) -> None:
             print(formatted_desc)
         print(f"\n\nActivated by {self.maybe_magenta(tinfo.provider)}")
         print("Valid fields:")
-        for field in sorted(tinfo.fields, key=lambda x: x.alias):
-            print()
```

Comment:
Does this extra newline mess up the markdown?

For terminal output, I prefer the extra space before a new section.

Reply:
Oh I am very sorry, my bad, deleted by accident. Have put it back now!